storage/store.go

```diff
@@ -921,7 +921,7 @@ func (s *Store) Start(stopper *stop.Stopper) error {
 				return true
 			}
 			wg.Add(1)
-			if !s.stopper.RunLimitedAsyncTask(sem, func() {
+			if err := s.stopper.RunLimitedAsyncTask(sem, func() {
```

Comment:
too much shadowing going on here. I think many of these would fare better as `if nil != ....` because the extra variable does more harm then good when it's not properly handled (tingling all kinds of spidey senses).